src/components/PricelistGeneralPanel.js

```diff
@@ -1,8 +1,17 @@
 import React from "react";
-import { withStyles, withTheme } from "@material-ui/core/styles";
+import { connect } from "react-redux";
 
+import { withStyles, withTheme } from "@material-ui/core/styles";
 import { Grid } from "@material-ui/core";
-import { FormPanel, withHistory, combine, withModulesManager, TextInput, PublishedComponent } from "@openimis/fe-core";
+
+import { FormPanel, withHistory, withModulesManager, PublishedComponent, ValidatedTextInput } from "@openimis/fe-core";
```

Comment:
I think this line could be quite too long. In case of such long expressions in imports you can write this in such way :
```
import { 
  FormPanel, withHistory, withModulesManager, 
  PublishedComponent, ValidatedTextInput 
} from "@openimis/fe-core";
```
In another project estlint converts such lines into something like this.

Reply:
the perfect example you have also in line 8th.

Reply:
Done.

Reply:
thx